tests/test_decode_attribute_types.py

```diff
@@ -123,6 +123,10 @@ def test_nominal(self):
         self.assertEqual(result[1][2], expected[1][2])
         self.assertEqual(result[1][3], expected[1][3])
 
+        # Check non-regression of ReDos raised in https://github.com/renatopp/liac-arff/issues/117
```

Comment:
Sorry, I don't see how this is checked at the moment, could you please explain this?

Reply:
Sorry, a couple of lines must have gone missing in commit.